sql/src/main/java/io/crate/analyze/symbol/Symbols.java

```diff
@@ -35,14 +35,16 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.function.Predicate;
+import java.util.function.UnaryOperator;
 
 public class Symbols {
 
-    public static final Predicate<Symbol> IS_COLUMN = s -> s instanceof Field || s instanceof Reference;
-
     private static final HasColumnVisitor HAS_COLUMN_VISITOR = new HasColumnVisitor();
+    private static final ReplacingSymbolVisitor<Void> DEEP_COPY_VISITOR = new ReplacingSymbolVisitor<>(ReplaceMode.COPY);
 
+    public static final Predicate<Symbol> IS_COLUMN = s -> s instanceof Field || s instanceof Reference;
```

Comment:
Wanted to do that, but there would be an illegal forward reference because the public DEEP_COPY uses a private constant